lib/resources/users.js

```diff
@@ -1,8 +1,10 @@
 module.exports = ({ api }) => {
   api.users = {
     get: (id, queryParams) => api.request(`users/${id}.json`, { queryParams }),
+    list: (queryParams) => api.request(`users.json`, { queryParams }),
```

Comment:
mmm we don't need those parenthesis, do we? `list: queryParams => api.request(`users.json`, { queryParams })`

Reply:
Just following the convention already there :)